server/base/src/main/java/org/apache/accumulo/server/master/state/TabletLocationState.java

```diff
@@ -93,6 +95,9 @@ public TServerInstance getServer() {
   }
 
   public TabletState getState(Set<TServerInstance> liveServers) {
+    if (suspend != null) {
```

Comment:
AFAICT the master will never set the suspend column when future is set.   In `MetaDataStateStore.suspend()` the suspend column is only set when `tls.current != null && suspensionTimestamp >= 0`